### PR TITLE
Switch python indentation from 2-space to 4-space

### PR DESCRIPTION
**Pull request by sbc100**
pep8 specifies 4 space indentation.  The use of 2 spaces is, I believe
a historical anomaly where certain large organizations such as google
chose 2 over 4 and have yet to make the switch.

Since there isn't too much code in binaryen today it seems reasonable to
make the switch.

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -1,6 +1,5 @@
 [flake8]
 ignore =
-  E111, # indentation not a multiple of 4 (we use 2))
   E114, # comment indentation not a multiple of 4
```
Comment (quantum5) [inline .flake8]: Should we remove this too?

Comment (sbc100) [reply .flake8]: Thanks done

File: .flake8 [hunk 2]
```diff
@@ -1,7 +1,5 @@
 [flake8]
 ignore =
-  E111, # indentation not a multiple of 4 (we use 2))
-  E114, # comment indentation not a multiple of 4
   E501, # line too long
   E121, # continuation line under-indented for hanging indent
```
Comment (quantum5) [inline .flake8]: Actually I think this is one also triggers when you use two-space indent for stuff hanging indent.

**Comment by kripken** [comment]
I don't have strong feelings on this. I feel that most python code I see is 2-space actually, but that's probably because it is from google or in codebases inspired by google code practices (like this one)... Do you have a sense of how common 4-space and 2-space are in the python ecosystem?

**Comment by quantum5** [comment]
4 space indentation is basically universal in the python ecosystem.

**Pull request merged**